src/pages/doyoungJung/Main/Main.js

```diff
@@ -1,8 +1,169 @@
 import React, { Component } from 'react';
+import InputFeed from './MainComponents/InputFeed';
+import Nav from '../../../components/Nav/Nav';
+import './Main.scss';
 
 class MainDoyoung extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feedData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData-Doyoung.json', {
```

Comment:
- `http://localhost:3000` 은 생략 가능합니다. 포트 번호가 달라지면(ex. `localhost:3001`) fetch 함수를 일일이 찾아서 코드를 수정을 해주어야 하기 때문에 작업이 복잡해 질 수 있습니다. 아래 코드 참고해서 수정해주세요!
```suggestion
    fetch('/data/feedData-Doyoung.json', {
```

Reply:
수정완료 🙆‍♂️  localhost는 생략 가능한 점 명심하겠습니다.